src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -899,7 +900,22 @@ private void CreateGroup(object parameters)
 
         private bool CanCreateGroup(object parameters)
         {
-            return DynamoSelection.Instance.Selection.OfType<ModelBase>().Any();
+            var groups = WorkspaceViewModel.Model.Annotations;
+            //Create Group should be disabled when a group is selected
+            if (groups != null && groups.Any(x => x.IsSelected))
+            {
```

Comment:
Why would groups ever be null?  Shouldn't it be guaranteed to be initialized when creating a `WorkspaceModel`?